5/writing-tests.md

```diff
@@ -0,0 +1,118 @@
+Writing Tests
+===
+
+Now that we have our test mock set up, we're ready to write some unit tests for our runtime.
+
+Let's start with a simple test that passes by default. Make sure to add a `#[test]` compiler flag followed by your test set up function. Let's use an `assert!(true)` inside the test for now to make it pass by default. 
+
+Don't forget that the test must execute within the context of `TestExternalities`!
+
+```rust
+#[test]
+fn it_works() {
+	with_externalities(&mut build_ext(), || {
+		assert!(true);
+	})
+}
+```
+
+The command for executing this test is: 
+`cargo test substratekitties.rs` or if want to test your entire package, simply run `cargo test -p YOUR_PACKAGE_NAME`.
```

Comment:
`cargo test substratekitties.rs` did not work for me.

`cargo test -p template-node-runtime` or whatever is what worked for me....

Is there some additional data we need to share to drive consistency here?

Reply:
On second thought, the correct way we should teach them is: 
`cargo test -p substratekitties-runtime substratekitties`
Which reads as:  run only the substratekitties unit test inside the substratekitties-runtime subpackage. 

My understanding of this is that we have to run the test from root directory in order to satisfy dependencies. But in root, `substratekitties-runtime` is a subpackage, so has to be specified. which is why in this particular case, we cannot just do `cargo test substratekitties.rs`